### PR TITLE
added an option to have split cells for notebooks

### PR DESCRIPTION
**Pull request by cdknorow**
updates to cell style, added ability to keep center and togle using a menu select item
updated markdown cell to also allow split window
switched order of cell styling

### CODE REVIEW

**Comment by ellisonbg** [comment]
To help us review this can you summarize the actual design changes you made
and post screen shots showing the changes?

On Sat, Apr 2, 2016 at 6:12 PM, cdknorow <notifications@github.com> wrote:

> updates to cell style, added ability to keep center and togle using a menu
> select item
> updated markdown cell to also allow split window
> switched order of cell styling
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/jupyter/notebook/pull/1297
> Commit Summary
>
>    - added quick hack for split screen notebook
>
> File Changes
>
>    - *M* notebook/static/notebook/js/actions.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-0> (16)
>    - *M* notebook/static/notebook/js/cell.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-1> (16)
>    - *M* notebook/static/notebook/js/codecell.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-2> (6)
>    - *M* notebook/static/notebook/js/menubar.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-3> (2)
>    - *M* notebook/static/notebook/js/notebook.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-4> (59)
>    - *M* notebook/static/notebook/js/textcell.js
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-5> (6)
>    - *M* notebook/templates/notebook.html
>    <https://github.com/jupyter/notebook/pull/1297/files#diff-6> (11)
>
> Patch Links:
>
>    - https://github.com/jupyter/notebook/pull/1297.patch
>    - https://github.com/jupyter/notebook/pull/1297.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/1297>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by ellisonbg** [comment]
Also, it looks like this PR has unrelated changes - can you rebase on master?

**Comment by takluyver** [comment]
Screenshot from gitter:

screenshot with split cell

I don't personally think this is a road we want to go down in the notebook itself, but maybe it would be useful as an extension.

**Comment by cdknorow** [comment]
Rebasing, so i deleted remote branch to start fresh.
